src/main/java/de/uhd/ifi/se/decision/management/jira/extraction/persistence/ActiveObjectsManager.java

```diff
@@ -756,7 +647,14 @@ private static boolean deleteAOElement(DecisionKnowledgeInCommentEntity elementT
 		}
 	}
 
+	/**
+	 * Proper way to delete sentences from ao. Also deletes their links
+	 * 
+	 * @param id
+	 * @return
+	 */
 	public static boolean deleteSentenceObject(long id) {
```

Comment:
Please try to use the same names as in the AbstractPersistenceInterface: deleteDecisionKnowledgeElement